src/ducks/connections/index.js

```diff
@@ -77,7 +82,9 @@ const reducer = (state = {}, action) => {
       }
 
       return action.response.data.reduce((newState, doc) => {
+        // const isAccount = isAccount(doc)
         const isTrigger = isKonnectorTrigger(doc)
+        // const creationInProgress = !!state.creation
```

Comment:
Any reason to keep these comented lines around?

Reply:
No reason at all, thanks.